proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java

```diff
@@ -331,12 +331,11 @@ public ProctorResult determineTestGroups(
                 ProctorUtils.convertToValueExpressionMap(
                         RuleEvaluator.EXPRESSION_FACTORY, inputContext);
 
-        final boolean evaluateAnonymousTestsOnly = getAnonymousEnabled(inputContext);
-
         for (final String testName : filteredEvaluationOrder) {
             final TestChooser<?> testChooser = testChoosers.get(testName);
             final String identifier;
-            if (evaluateAnonymousTestsOnly && !testChooser.getTestDefinition().getIncognito()) {
+            if (getIncognitoEnabled(inputContext)
```

Comment:
NIT, Is the method name `isIncognitoEnabled(inputContext)` better?

Reply:
Renamed